index.js

```diff
@@ -20,13 +21,15 @@ const {
 const {
   validateArray,
   validateBoolean,
+  validateBooleanArray,
   validateObject,
   validateString,
+  validateStringArray,
   validateUnion,
 } = require('./internal/validators');
 
 const {
-  kEmptyObject,
+  kEmptyObject
```

Comment:
Dangling commas is a pattern being used in nodejs code. It means the line does not change it another item is added to the object, so the git diff and line history is cleaner.

It does look a big odd on this single line, but you will see the dangling comma on all the other require blocks too.